config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
+  resources :activities, except: [:new, :edit]
   root to: 'air_quality_observations#index'
   namespace :api do
     resources :air_quality_observations, except: [:new, :edit]
+    resources :activities, only: [:post, :index]
```

Comment:
This is neat syntax; I've always done `only: [:create, :index]` but if it recognizes `:post` then :+1: